### ISSUE TITLE
Make it repair itself after being relocated, or at least ask for a local .ckpt file

### ISSUE THREAD

**Issue by segersj2** [cmdr2/stable-diffusion-ui#278]
I installed in my downloads folder to start with, decided I wanted to keep it, but when I moved it to a new folder (/home/<me>/Local-apps/SD-with-queing) it broke, and I tried to fix it, and now it's really broken and I'm going to reinstall. My plan is to kill the installer when it starts the .ckpt download, sub in the one I already have. My plan worked.

Ideally, I would like it to use relative paths, failing that, I would like it to fix itself after a move. Failing that, I would like a warning in the readme that this is a known issue and will not be fixed, put it in its final resting place before running start.sh the first time.

All that said, thank you for this wonderful program, I'm so grateful. :1st_place_medal:

**Comment by segersj2** [comment]
I grepped for the path and got back something like 4000 places the path is hard coded with an absolute path, I grepped the results for ".conda" and that narrowed it to 783 possibilities. I grepped that for "env" and narrowed it to 540 results. I'm afraid this is beyond me. 

Thank you for the effort, I do appreciate it

**Comment by segersj2** [comment]
./open_dev_console needed to be made executable, no problem. next issue: conda is not in $PATH, so I found it in installer/bin/conda, and ran ./installer/bin/conds config --add pkgs_dirs <path to directory>/pkgs (properly replaced with actual path). Conda replied: 
```
Traceback (most recent call last):
  File "/home/jj/Downloads/SD-with-queing/stable-diffusion-ui/./installer/bin/conda", line 12, in <module>
    from conda.cli import main
ModuleNotFoundError: No module named 'conda'
```
So I tried "cd"ing into installer/bin and calling it from there, same result.

HOWEVER, Whatever you changed, makes it work flawlessly from the start.sh - it Just Works!
I'll be happy to help out, but it seems you've already fixed it, I tried moving it two times.
